scripts/databases/seed.sh

```diff
@@ -3,23 +3,44 @@
 set -euo pipefail
 
 GROUP=$1
+shift
+DATABASE_VERSIONS=$*
```

Comment:
Kinda nit pick but maybe remove the "DATABASE_" prefix. It should be clear from the context but I really don't feel strong about this one. It just came to mind when I saw [this](https://github.com/dhis2-sre/im-manager/pull/306/files#diff-5f9562bf7fb13c1dedb12edd181ae94501e089f2b309987a684e2be742a86f4bR42) line.